ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+----
+./bin/jboss-cli.sh "embed-server --server-config=standalone-ha.xml,\
+  xa-data-source add --name=ejbJtaDs --driver-name=postgresql --jndi-name=java:jboss/datasources/ejbJtaDs --user-name=test --password=test --xa-datasource-properties=ServerName=localhost,\
+  /subsystem=datasources/xa-data-source=ejbJtaDs/xa-datasource-properties=PortNumber:add(value=5432),\
+  /subsystem=datasources/xa-data-source=ejbJtaDs/xa-datasource-properties=DatabaseName:add(value=test)"
+----
+
+NOTE: For Windows, use the `bin\jboss-cli.bat` script.
+
+[#_start_productname_servers]
+== Start {productName} servers
+
+You need to have three servers running. The first one `server1` is to be started
+with the `standalone.xml` configuration.
+The other ones (the `server2` and the `server3`) are to be started
+with the `standalone-ha.xml` configuration.
+
+If you are going to start the servers on the local machine you may consider
+to use port offset for each server binds to a different port.
+Then it's mandatory to define unique transaction node id
+and jboss node name for each server.
+This can be done with using the system properties `jboss.tx.node.id` and `jboss.node.name`
+on the server startup.
+
+Run these commands each in a separate terminal.
+
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
+----
+cd ${jbossHomeName}_1
+./bin/standalone.sh -c standalone.xml -Djboss.tx.node.id=server1 -Djboss.node.name=server1 -Dwildfly.config.url=${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/configuration/custom-config.xml
+
+cd ${jbossHomeName}_2
+./bin/standalone.sh -c standalone-ha.xml -Djboss.tx.node.id=server2 -Djboss.node.name=server2 -Djboss.socket.binding.port-offset=100
+
+cd ${jbossHomeName}_3
+./bin/standalone.sh -c standalone-ha.xml -Djboss.tx.node.id=server3 -Djboss.node.name=server3 -Djboss.socket.binding.port-offset=200
+----
+
+NOTE: For Windows, use the `bin\standalone.bat` script.
+
+[#_running_the_quickstart]
+== Running the Quickstart
+
+. Start the {productName} servers as described above
+. Clean and build the project by navigating to the root directory of this quickstart in terminal and run
++
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/
+mvn clean install
+----
++
+. Navigate to the subfolder `client` of this quickstart `ejb-txn-remote-call`
+and deploy `war` application to the `server1` with running
++
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client
+mvn wildfly:deploy
+----
++
+. Navigate to the subfolder `server` of this quickstart `ejb-txn-remote-call`
+and deploy `war` application to the `server2` and `server3` with running
++
+[source,sh,subs="+quotes,attributes+",options="nowrap"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server
+mvn wildfly:deploy -Dwildfly.port=10090
+mvn wildfly:deploy -Dwildfly.port=10190
+----
+
+The commands should be finished without any error.
```

Comment:
```suggestion
The commands should finish without any errors.
```
What errors might occur? What should the user do to correct problems that cause errors?